plugins/katello/developers.md

```diff
@@ -45,53 +42,56 @@ There are a few important states that issues can be in that are worth knowing ab
   * Needs design: The issue requires investigation by a developer
   * Ready for Testing: An issue has an open PR
   * Closed: An issue has been fixed
+  * Rejected: An issue needs more information, but the author has not responded within two weeks
+
+If Needs More Information is set during triage, the issue's triaged state should be left alone for review the following week. If there is no reponse on the issue after two weeks, the runner should set the state to Rejected, leaving the issue's triaged state alone, with the following comment:
+
+> Please reopen this issue with the requested information, and we will be happy to take a look.
 
 ### Release States
 
 There are a few important states the issue can be in with respect to the 'Release' field within Redmine. These release states have an impact not only a release itself but the triage process as a whole. A quick definition of each that will make more sense in the process section:
 
-  * Empty: No release has been set, this is the "untriaged" bucket
-  * Katello Backlog: Issues that have been put on the backlog and not scheduled for a release due to any number of reasons
-  * Katello Recycle Bin: Issues that have been rejected, or are a duplicate, or have been in 'Needs more information' for longer than a week
+  * Empty: No release has been set. Triaged issues with empty release states are implied backlog (open issues) or recycle bin (closed issues).
```

Comment:
Are Backlog and Recycle Bin gone?

Reply:
Yes

Reply:
Wait, no, they're still there, but we're not going to use them anymore.

Reply:
Can you explain why they are being dropped?

Reply:
So triaged, closed represents Recycle Bin? How are Recycle Bin candidate issues represented? 

The part I dislike is "implied backlog". I personally liked that there was no implication before. The backlog was obvious and we are losing that now. I feel that's less obvious to users and even developers.

Reply:
Yes, I agree it's less obvious. @jlsherrill !?!? ;)

Reply:
I think we should keep the recycle bin and backlog targets and use them like we have been so it's not confusing.